### PR TITLE
add-component

### PR DESCRIPTION
**Pull request by BasitKorede**
# Add App Component

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches the given design

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          <button type="button">1</button>
+          <button type="button">2</button>
+          <button type="button">3</button>
+          <button className="highlight" type="button">+</button>
+          <button className="bottom" type="button">0</button>
+          <button className="bottomII" type="button">.</button>
+          <button className="highlight bottomIII" type="button">=</button>
+        </div>
+      </div>
+    );
```
Comment (Gambit142) [inline src/components/Calculator.js:40]: - [ ] (Optional) You can consider putting the button symbols in an array and then map through the array to render each button. That way, your code remains _**DRY**_. Kindly read this [article](https://www.pluralsight.com/guides/how-to-use-the-map()-function-to-export-javascript-in-react) to get a better understanding of what I am suggesting.

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @BasitKorede 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Created a components directory 💯 
- Created class-based React components `App` and `Calculator` 💯 
- Your calculator matches the design of the template given 💯 

## Required Changes ♻️

- Remember you are to delete all boilerplates that were created with the _**create react app**_ command. In light of this, kindly delete the following files:
  -  App.test.js
  - logo.svg
  - reportWebVitals.js
  - setupTests.js
  - public/manifet.json
  - robots.txt

- In your `REAME.md` file, the URL for cloning this repository is wrong, kindly correct it. Doing this would make your `README.md` file look professional and it indicates that you pay attention to details. 
- Kindly add your name as the author of this repository and include any social media handles you wish. Remember that you are the sole author of this repository and that means you don't need a second author. 👍 

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- You can also consider removing this section from your `README.md` since you don't have instructions under these sections

image

- You can consider adding instructions for building the project `npm run build` and for starting the project on a local browser `npm run start`. 👍 

- You can also consider the optional suggestion I made under the review. 👍 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.js (hunk 1 above)
Comment (BasitKorede) [reply src/components/Calculator.js:40]: Hello @Gambit142 Thanks for the review. Changes made .

**Comment by arslanbisharat** [comment]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**